<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDataEditDialog</class>
 <widget class="QDialog" name="BaseDataEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基础数据库访问窗口</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>681</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mLayout" stretch="1,2,5,1,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>请在下面输入要执行的sql语句：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExec">
        <property name="text">
         <string>执行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbTables"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnSwitch">
        <property name="text">
         <string>只读模式</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="edtSql"/>
    </item>
    <item>
     <widget class="QTableView" name="tview">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSubmit">
        <property name="text">
         <string>提交更改</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRevert">
        <property name="text">
         <string>取消更改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>添加记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>删除记录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblMsg">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExec</sender>
   <signal>clicked()</signal>
   <receiver>BaseDataEditDialog</receiver>
   <slot>excuteSql()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>excuteSql()</slot>
 </slots>
</ui>
